packages/vitest/src/node/pools/threads.ts

```diff
@@ -107,7 +107,7 @@ export function createThreadsPool(ctx: Vitest, { execArgv, env }: PoolProcessOpt
       catch (error) {
         // Worker got stuck and won't terminate - this may cause process to hang
         if (error instanceof Error && /Failed to terminate worker/.test(error.message))
-          ctx.state.addProcessTimeoutCause(`Failed to terminate worker while running ${files.join(', ')}.`)
+          ctx.state.addProcessTimeoutCause(`Failed to terminate worker while running ${files.join(', ')}. \nYou might want to try running with "--pool=forks" option.`)
```

Comment:
Should we also include link to the issue?

```suggestion
          ctx.state.addProcessTimeoutCause(`Failed to terminate worker while running ${files.join(', ')}. \nYou might want to try running with "--pool=forks" option. \nSee https://github.com/vitest-dev/vitest/issues/3077 for more information.`)
```

Reply:
This message might not make much sense to users unless "why" is explained, so providing an issue link (or even with exact comment of yours like https://github.com/vitest-dev/vitest/issues/3077#issuecomment-1815767839) feels nicer to me.

I don't know much about this issue as I haven't seen nor investigated myself, but if it's likely that the issue is due to `worker_threads` compatibility of some external factors (either user's dependency or nodejs/undici fetch), then this is more like user's wrong usage of Vitest, so maybe we can add this error to https://vitest.dev/guide/common-errors.html and guide users to "forks" with some explanation?

Reply:
Good points. Maybe this should simply be a `Failed to terminate worker while running some.test.ts. See https://vitest.dev/guide/common-errors.html#failed-to-terminate-worker` error, and on that page list more information about common causes.